src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/SubmissionParameters.java

```diff
@@ -0,0 +1,250 @@
+package org.fogbowcloud.saps.engine.core.dispatcher;
+
+import java.util.Date;
+
+/**
+ * DTO of parameters from a {@link Submission}.
+ */
+public class SubmissionParameters {
```

Comment:
Do we need all these set methods? are they here because of this object is created by reflection? (say, similar to springboot stuff)?

Reply:
No, one again I tried to maintain the pattern found in other models (e.g. ImageTask.java), but I can make the fields public and remove the getters and setters. Which approach do you think it's better?

Reply:
I think it is better to keep the code as small as possible. So, I'd vote to keep the object immutable. The fields should be private final, remove the set methods and keep the get ones.

Reply:
I understand, makes sense to me. I'll do it as you suggested